src/window.rs

```diff
@@ -151,37 +196,25 @@ fn retransmit_all_leader_blocks(
     consumed: u64,
     received: u64,
     retransmit: &BlobSender,
+    window: &SharedWindow,
 ) -> Result<()> {
-    let mut retransmit_queue = VecDeque::new();
+    let mut retransmit_queue: VecDeque<SharedBlob> = VecDeque::new();
```

Comment:
the type had to be added?

Reply:
Yes, as it's being passed as parameter to add_block_to_retransmit_queue(). Not sure if there's a better way.

Reply:
@garious or Rust beginners might have advice